rust/actyx/ax-core/build.rs

```diff
@@ -1,3 +1,26 @@
+use std::process::Command;
+
+fn git_hash() -> String {
+    // https://stackoverflow.com/a/5737794/576180
+    let out = Command::new("git")
+        .arg("status")
+        .arg("--porcelain")
+        .output()
+        .expect("Error running git status --porcelain");
```

Comment:
We can’t expect this for everyone running `cargo install ax`, and it would be meaningless in that context anyway. I’d suggest injecting this environment variable in github actions, not from build.rs.

Reply:
And on missing we keep it empty or maybe tag the build like "cargo_install"?

Reply:
`ax 2.18.0-cargo_install-linux-x86_64-release` looks good to me